interface/templates/interface/upload.html

```diff
@@ -13,6 +13,12 @@
         {% include "interface/tracker.html" %}
     </head>
 
+    {% if delta_t %}
+        <script type="text/javascript">
+            alert('Please wait {{delta_t}}s before doing another submission');
```

Comment:
I suggest that, instead of throwing a popup in the face of the user, we show a [bootstrap alert](https://getbootstrap.com/docs/4.4/components/alerts/).